commands/channel_test.go

```diff
@@ -2176,3 +2176,268 @@ func (s *MmctlUnitTestSuite) TestRenameChannelCmd() {
 		s.Require().Equal(printer.GetLines()[0], updatedChannel)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestCreateChannelCmd() {
+	s.Run("should not create channel without display name", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		teamName := "teamName"
+		channelName := "channelName"
+		argsTeamChannel := teamName + ":" + channelName
+		args := []string{""}
+		args[0] = argsTeamChannel
```

Comment:
I think we can safely use this instead:
```suggestion
		args := []string{teamName + ":" + channelName}
```